cassandane/Cassandane/Cyrus/Fetch.pm

```diff
@@ -55,7 +55,12 @@ $Data::Dumper::Sortkeys = 1;
 sub new
 {
     my $class = shift;
-    return $class->SUPER::new({ adminstore => 1 }, @_);
+
+    my $config = Cassandane::Config->default()->clone();
+    $config->set(jmap_preview_annot => '/shared/vendor/messagingengine.com/preview',
+    );
```

Comment:
nit: Weird closing parent placement. Would put this all on one line or do this if it needs to fit in 80:
```
    $config->set(
        jmap_preview_annot => '/shared/vendor/messagingengine.com/preview',
    );

```

Reply:
Hmm.  Quick cut-n-paste from another test and I didn't look at the formatting.  I will fix

Reply:
Fixed